app/src/main/java/com/kickstarter/features/pledgedprojectsoverview/ui/PledgedProjectsOverviewActivity.kt

```diff
@@ -137,6 +138,16 @@ class PledgedProjectsOverviewActivity : AppCompatActivity() {
                                     env.analytics()?.trackPPOOpenSurveyCTAClicked(PPOCard.projectId ?: "", ppoCardPagingSource.itemSnapshotList.items, totalAlerts, PPOCard.surveyID ?: "")
                                     openBackingDetailsWebView(
                                         url = PPOCard.backingDetailsUrl ?: "",
+                                        toolbarTitle = R.string.Backing_details,
+                                        resultLauncher = startForResult
+                                    )
+                                }
+
+                                PPOCardViewType.PLEDGE_MANAGEMENT -> {
+                                    env.analytics()?.trackPPOFinalizePledgeCTAClicked(PPOCard.projectId ?: "", ppoCardPagingSource.itemSnapshotList.items, totalAlerts)
```

Comment:
nitpick: 
you could call the analytics event from the VM and add tests if you like.

There is a somewhat similar example on LatePledges ->
UI -> https://github.com/kickstarter/android-oss/blob/e270ee9d09da5555927f50765da31581f9909b76/app/src/main/java/com/kickstarter/ui/activities/ProjectPageActivity.kt#L679

VM -> https://github.com/kickstarter/android-oss/blob/e270ee9d09da5555927f50765da31581f9909b76/app/src/main/java/com/kickstarter/viewmodels/projectpage/LatePledgeCheckoutViewModel.kt#L449

Test -> https://github.com/kickstarter/android-oss/blob/e270ee9d09da5555927f50765da31581f9909b76/app/src/test/java/com/kickstarter/viewmodels/LatePledgeCheckoutViewModelTest.kt#L673